snapshot-labs/sx-monorepo
# refactor: use AppLink to render button like link

### Summary



Depends on #759 

This PR will make use of the newly added AppLink component to render links looking like a button properly

This will avoid having to nest a button inside a link, just to have the button UI, and get rid of the `tabindex=-1` hack to avoid double focus on keyboard navigation

This also enable back the link preview on focus like:

![Screenshot 2024-09-15 at 00 51 08](https://github.com/user-attachments/assets/92b7fc27-049f-4c65-a64b-d76dc4aea856)


### How to test

1. Go to landing page
2. All links like button should render same as before
3. Inspect the element, the `<a>` element should have the classes to look like a button, and there's not nested button anymore
